plugins/hypervisors/kvm/src/main/java/com/cloud/hypervisor/kvm/resource/wrapper/LibvirtMigrateCommandWrapper.java

```diff
@@ -338,7 +382,7 @@ private String replaceStorage(String xmlDesc, Map<String, MigrateCommand.Migrate
                         String path = getPathFromSourceText(migrateStorage.keySet(), sourceText);
 
                         if (path != null) {
-                            MigrateCommand.MigrateDiskInfo migrateDiskInfo = migrateStorage.remove(path);
+                            MigrateCommand.MigrateDiskInfo migrateDiskInfo = migrateStorage.get(path);
 
```

Comment:
Was this a bug, to get and not remove a path. Can you send any bugfix PR towards 4.11?

Reply:
This `remove` affects file migration (local file or NFS); however, before this, there was no KVM storage migration considering file path. Therefore, it does not impact on previous releases.